src/components/EmojiPicker/EmojiPicker.vue

```diff
@@ -307,8 +323,9 @@ export default {
 			flex-basis: calc(100% / 8);
 			text-align: center;
 
-			&:hover::before {
-				background-color: var(--color-background-hover);
+			&:hover,
+			&.emoji-mart-emoji-selected{
+				border: 1px solid var(--color-primary);
```

Comment:
```suggestion
			&:hover::before,
			&.emoji-mart-emoji-selected::before {
				background-color: var(--color-background-hover);
```

Reply:
yes, i see. 
If we leave it like this, we won't have an outline when we use the arrow keys. That was what I was trying to achieve.

Reply:
might be a chrome thing that i do not see the hover effect. I ca see it in firefox ...

Reply:
I can see it in chrome :)

Reply:
... oh no. it is my monitor! never detected this before. I can't see the slightest change in any browser.  (thought i was going crazy for a second).
ok. take back my changes. sry for the confusion.